scripts/workers/submission_worker.py

```diff
@@ -212,22 +213,21 @@ def extract_challenge_data(challenge, phases):
     EVALUATION_SCRIPTS[challenge.id] = challenge_module
 
 
-def load_active_challenges():
+def load_active_challenge(challenge_pk):
```

Comment:
Let's rename this method to `load_challenge` since we only load a challenge now and not check whether it is active or not.